src/components/correct-name/CorrectNameRequest.vue

```diff
@@ -198,10 +198,11 @@ export default class CorrectNameRequest extends Mixins(NameRequestMixin) {
           this.emitSaved(true)
         }
       } catch (error) {
-        alert((error as any).message)
         // inform parent that process is complete
         this.$refs.form.resetValidation()
         this.emitSaved(false)
+
+        alert((error as any).message)
```

Comment:
At the test, TypeError: alert is not a function.
The test was not able to run `this.emitSaved(false)`

So, I changed code like this for test case

Reply:
Could you mock `alert` instead, and check that it has been called?